lib/driver.js

```diff
@@ -176,23 +179,32 @@ export class Driver {
             this.log("Bad addr config version: %s", pkt.version);
             return false;
         }
-        
+
         const parsed = this.handleAddrs(pkt.addrs);
         if (!parsed) return false;
 
-        const specs = parsed.map(([, s]) => s);
-        if (!await this.subscribe(specs))
-            return false;
```

Comment:
We need to run `subscribe` here too, as we can be sent a new list of addresses. (The handler's subscribe function will need to handle unsubscribing if necessary...) This can also be fire-and-forget, at the end (after we set `this.addrs`).